collect-monthly.sh

```
@@ -0,0 +1,174 @@
+    fi
+    
+    jq "${jq_args[@]}" '
+        reduce (inputs | .[]) as $item ([]; 
+            . + if any(.[]; .url == $item.url) then [] else [$item] end
+        )
+    '
+}
+
+# Update all_prs.json with unique PRs from all monthly files
+echo "Updating all_prs.json..."
+find data/monthly -name "prs_*.json" -type f | xargs cat | \
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)